src/consumer/runner.js

```diff
@@ -187,6 +187,7 @@ module.exports = class Runner {
         },
         uncommittedOffsets: () => this.consumerGroup.uncommittedOffsets(),
         isRunning: () => this.running,
+        isStale: ({ topic, partition }) => this.consumerGroup.hasSeekOffset({ topic, partition }),
```

Comment:
Maybe `isStale` isn't clear enough, what do you think? I can't offer a better name now, let me think about this.

Reply:
Naming this is always hard :/ I tried to reason from a person implementing `eachBatch` and how the name of the method should hint at what they can or cannot do. In this case we'd want to imply that one _could_ continue processing, but that messages have gone outdated. Maybe `isCanceled` makes more sense?

Reply:
Sitting on `isCanceled` for a bit, I think it might be confusing in combination with `isRunning`. What would be the difference between the two? While `isStale` in isolation is maybe less obvious, I do think it avoids that confusion.